File: Leetcode/389-Find-the-Difference/FindTheDifference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        char cur = t.at(0);
        for (int i=1; i<t.length(); i++) {
            cur = (char)(cur^t.at(i));
        }
        for (int i=0; i<s.length(); i++) {
            cur = (char)(cur^s.at(i));
        }
        return cur;
    }
};

File: Leetcode/389-Find-the-Difference/FindTheDifference.java
public class FindTheDifference {
    public char findTheDifference(String s, String t) {
        char cur = t.charAt(0);      
        for (int i=1; i<t.length(); i++) {
            cur = (char)(cur^t.charAt(i));
        }
        for (int i=0; i<s.length(); i++) {
            cur = (char)(cur^s.charAt(i));
        }
        return cur;
    }// end of findTheDifference
}// end of Solution 





